extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/BarrageUtil.java

```diff
@@ -425,6 +426,11 @@ private static ConvertedArrowSchema convertArrowSchema(
             if (type.getValue() == null) {
                 Class<?> defaultType = getDefaultType(getArrowType.apply(i), result, i);
                 type.setValue(defaultType);
+            } else if (type.getValue() == boolean.class) {
+                // check existing barrage clients that might be sending int8 instead of bool
+                // TODO (deephaven-core#3404) widen this check for better assurances
```

Comment:
Wrong ticket number:
```suggestion
                // TODO (deephaven-core#3403) widen this check for better assurances
```